src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represent a parent class for a Person's contacts
+ */
+public class Contact {
+	private String value;
+	private boolean isPrivate;
+	
+	public Contact() {}
+	
+	public void setValue(String value) {
+		this.value = value;
+	}
+	
+	public String getValue() {
+		return this.value;
+	}
+	
+	public void setPrivate(boolean isPrivate) {
+		this.isPrivate = isPrivate;
+	}
+	
+	
+	
+	@Override
+	public String toString() {
+		return value;
+	}
+
+	@Override
+	public int hashCode() {
```

Comment:
hashCode and equals() methods go hand in hand. So always bring the hashCode() and the equals() methods together.